board/safety.h

```diff
@@ -111,6 +113,10 @@ uint16_t current_safety_param = 0;
 static const safety_hooks *current_hooks = &nooutput_hooks;
 safety_config current_safety_config;
 
+static bool is_lat_active(void) {
+  return controls_allowed || mads_is_lateral_control_allowed_by_mads();
```

Comment:
because now we have `controls_allowed` available on `safety_mads.h` how about we move this comparison to `safety_mads.h` code? 

```cpp
inline bool mads_is_lateral_control_allowed_by_mads(void) {
  return m_mads_state.system_enabled && m_mads_state.controls_allowed_lat || *m_mads_state.op_controls_allowed.current;
}
```


```suggestion
  // This is just illustrative. We'd call this instead of is_lat_active(). 
  return mads_is_lateral_control_allowed_by_mads();
```

Reply:
Let's keep the name more generically inside `safety.h` since it will be used universally for steering tx.